File: src/main/kotlin/ru/csdeep/model/ApplicationSetting.kt
package ru.csdeep.model

/**
 * DataBaseSetting data object.
 */
data class DataBaseSetting(
    val serveName: String,
    val databaseName: String
)

/**
 * AppSettings data object.
 */
data class ApplicationSetting(
    val dataBaseSetting: DataBaseSetting
)

File: src/webapp/src/model/ApplicationSetting.ts
export type DataBaseSetting = {
    serveName: String
    databaseName: String
}

export type ApplicationSetting = {
    dataBaseSetting?: DataBaseSetting
}

export interface ErrorMessageResponse {
    timestamp?: string
    status?: number
    error?: string
    exception?: string
    trace?: string
    message?: string
    path?: string
}